Makefile

```diff
@@ -0,0 +1,58 @@
+LIGHTTPD_VERSION = 1.4.77
+OUTFILE = lighttpd
```

Comment:
shouldn't that rather be an absolute path (for the sake of "readability", preferably passed from caller (in Dockerfile))?